io.c

```diff
@@ -7374,6 +7374,9 @@ static VALUE popen_finish(VALUE port, VALUE klass);
  *  Executes the given command +cmd+ as a subprocess
  *  whose $stdin and $stdout are connected to a new stream +io+.
  *
+ *  This method has potential security vulnerabilities;
+ *  see {Command Injection}[doc/command_injection_rdoc.html].
```

Comment:
Is the `doc/command_injection.rdoc` file supposed to be added in this commit?

Reply:
Added.